Cargo.toml

```diff
@@ -564,6 +566,9 @@ signature = "2.1.0"
 sec1 = "0.7.0"
 parking_lot = "0.12.0"
 paste = "1.0.7"
+passkey-authenticator = { git = "https://github.com/aptos-labs/passkey-rs", branch = "fix-passkey-rs", features = ["testable"] }
+passkey-client = { git = "https://github.com/aptos-labs/passkey-rs", branch = "fix-passkey-rs" }
+passkey-types = { git = "https://github.com/aptos-labs/passkey-rs", branch = "fix-passkey-rs" }
```

Comment:
Why are you importing the same thing twice here? `passkey-client` and `passkey-types` seem to be the same dependency.

Reply:
I don't quite follow. I believe they're all different packages? 
- [passkey-client](https://crates.io/crates/passkey-client)
- [passkey-types](https://crates.io/crates/passkey-types)

Reply:
Oh, I see :facepalm: